docs/UserGuide.md

```diff
@@ -404,11 +404,91 @@ Examples:
 * `importcsv n/2`
   * Reads from CSV treating the 2nd row as names, 3rd row as phone numbers, 4th row as email address, 5th row as tagged.
 
-
 ### Archiving data files `[coming in v2.0]`
 
 _Details coming soon ..._
 
+###Extension on Find contact: `find` `t/`
+
+Locates persons by different search types
+
+Format: `t/SEARCH_TYPE k/KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. hans will match Hans
+* The order of the keywords does not matter. e.g. Hans Bo will match Bo Hans
+* `SEARCH_TYPE` must match one of the following:
+  * Name
+  * Address
+  * Phone
+  * Email
```

Comment:
We should make this an optional field, and default to search by name if not specified.
```suggestion
* `SEARCH_TYPE` must match one of the following:
  * Name
  * Address
  * Phone
  * Email
* If `SEARCH_TYPE` is not specified, default search type is by `Name`.
```